SPCTR-3222: Improvement: Container background overlay styles (#3400)
brainstormforce/ultimate-addons-for-gutenberg: README.md

## Changelog ##

### x.x.x ###

> * Improvement: Container - Added an opacity setting for the overlay of the image and video background-types.

Reviewer: ```suggestion
* Improvement: Container - Added an image overlay option for the background.
```

Author: Updated.
